cpp/src/io/orc/orc.h

```diff
@@ -197,23 +197,21 @@ class ProtobufReader {
     return (field_number * 8) + PB_TYPE_VARINT;
   }
 
-  template <typename base_t,
-            typename std::enable_if_t<std::is_same<base_t, float>::value>* = nullptr>
+  template <typename base_t, typename std::enable_if_t<std::is_same_v<base_t, float>>* = nullptr>
   int static constexpr encode_field_number_base(int field_number) noexcept
   {
     return (field_number * 8) + PB_TYPE_FIXED32;
   }
 
-  template <typename base_t,
-            typename std::enable_if_t<std::is_same<base_t, double>::value>* = nullptr>
+  template <typename base_t, typename std::enable_if_t<std::is_same_v<base_t, double>>* = nullptr>
   int static constexpr encode_field_number_base(int field_number) noexcept
   {
     return (field_number * 8) + PB_TYPE_FIXED64;
   }
 
   template <typename T,
-            typename std::enable_if_t<!std::is_class<T>::value or
-                                      std::is_same<T, std::string>::value>* = nullptr>
+            typename std::enable_if_t<!std::is_class<T>::value or std::is_same_v<T, std::string>>* =
```

Comment:
There might be value in a follow-up PR to convert these SFINAE checks into `CUDF_ENABLE_IF()`.

Reply:
Once we get clang-tidy set up we could definitely automate this.